scripts/rnnlm/lmrescore_pruned.sh

```diff
@@ -17,6 +17,7 @@ max_ngram_order=4 # Approximate the lattice-rescoring by limiting the max-ngram-
                   # exploding exponentially. Details of the n-gram approximation
                   # method are described in section 2.3 of the paper
                   # http://www.danielpovey.com/files/2018_icassp_lattice_pruning.pdm
+max_arcs=         # limit the max arcs in lattice while rescoring
```

Comment:
can you please make a separate PR for this?  or have Ke or Hainan do it?

Reply:
Sure. A PR to master or material_basic branch?